lib/IRGen/IRGenModule.cpp

```diff
@@ -1444,6 +1444,11 @@ void IRGenModule::constructInitialFnAttributes(
     Attrs.addAttribute(llvm::Attribute::StackProtectReq);
     Attrs.addAttribute("stack-protector-buffer-size", llvm::utostr(8));
   }
+
+  if (Context.LangOpts.hasFeature(Feature::Embedded) &&
+      !Context.LangOpts.EnableCXXInterop) {
+    Attrs.addAttribute(llvm::Attribute::NoUnwind);
```

Comment:
Is that applied only to Swift functions, and not the imported C++ functions? In this case you can enable `nounwind` for Swift functions, as exceptions are always caught at the boundary between Swift and a call to C++.

Reply:
I think you're right, this only applies to SILFunctions, not imported C++ functions. @eeckstein is that correct?